### ISSUE TITLE
Legion 16irx8h Sleep Issue

### ISSUE THREAD

**Issue by MAkcanca** [NixOS/nixos-hardware#1043]
**System Information**

    Device: Lenovo Legion 16irx8h
    Kernel Version: 6.9.8
    OS: NixOS with mostly default setup
    Hardware Configuration: Using nixos-hardware/lenovo/legion/16irx8h

**Problem Description**

When waking the laptop from sleep, it successfully resumes and prompts for login. However, after logging in, the system automatically goes back to sleep within approximately 5 seconds. The screen turns off, and the power button starts blinking slowly, indicating sleep mode. This behavior is consistent and repeatable.
**Steps to Reproduce**

1.     Put the laptop to sleep.
2.     Wake the laptop from sleep.
3.     Login when prompted.
4.     Observe the laptop going back to sleep after a few seconds.

**Expected Behavior**

After logging in, the laptop should remain awake and operational.
**Additional Information**

-     This issue occurs consistently after each wakeup from sleep.
-     No additional configurations have been made beyond the default setup for the hardware and kernel.

**Comment by MAkcanca** [comment]
`journalctl -b` logs, last 400 lines

```
Jul 18 15:45:42 nixos dockerd[2853]: time="2024-07-18T15:45:42.575700888+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:55916->1.1.1.1:53: i/o timeout"
Jul 18 15:45:45 nixos .tracker-miner-[2692]: Could not launch metadata extractor: Failed to execute child process “/nix/store/gq81ix20x1qm7nqjpd9cdy5bd9n9x3yn-tracker-miners-3.7.3/libexec/tracker-extract-3” (Permission denied)
Jul 18 15:45:49 nixos .tracker-miner-[2692]: Could not launch metadata extractor: Failed to execute child process “/nix/store/gq81ix20x1qm7nqjpd9cdy5bd9n9x3yn-tracker-miners-3.7.3/libexec/tracker-extract-3” (Permission denied)
Jul 18 15:45:52 nixos systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Jul 18 15:45:55 nixos systemd-logind[1584]: The system will suspend now!
Jul 18 15:45:55 nixos ModemManager[2879]: <msg> [sleep-monitor-systemd] system is about to suspend
Jul 18 15:45:55 nixos dbus-daemon[1528]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.11' (uid=0 pid=1675 comm="/nix/store/3kimhm61nra85879sxqqg7ilsm200zwi-networ" label="kernel")
Jul 18 15:45:55 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:45:55 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:45:55 nixos systemd[1]: Starting Network Manager Script Dispatcher Service...
Jul 18 15:45:55 nixos dbus-daemon[1528]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Jul 18 15:45:55 nixos systemd[1]: Started Network Manager Script Dispatcher Service.
Jul 18 15:45:55 nixos kernel: wlp0s20f3: deauthenticating from 5c:64:8e:64:3b:12 by local choice (Reason: 3=DEAUTH_LEAVING)
Jul 18 15:45:55 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-DISCONNECTED bssid=5c:64:8e:64:3b:12 reason=3 locally_generated=1
Jul 18 15:45:55 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-DSCP-POLICY clear_all
Jul 18 15:45:55 nixos avahi-daemon[1520]: Interface wlp0s20f3.IPv4 no longer relevant for mDNS.
Jul 18 15:45:55 nixos avahi-daemon[1520]: Leaving mDNS multicast group on interface wlp0s20f3.IPv4 with address 192.168.1.42.
Jul 18 15:45:55 nixos avahi-daemon[1520]: Joining mDNS multicast group on interface wlp0s20f3.IPv4 with address 192.168.1.42.
Jul 18 15:45:55 nixos avahi-daemon[1520]: New relevant interface wlp0s20f3.IPv4 for mDNS.
Jul 18 15:45:55 nixos systemd[1]: Stopped target Host and Network Name Lookups.
Jul 18 15:45:55 nixos systemd[1]: Stopping Host and Network Name Lookups...
Jul 18 15:45:55 nixos systemd[1]: Stopped target User and Group Name Lookups.
Jul 18 15:45:55 nixos systemd[1]: Stopping User and Group Name Lookups...
Jul 18 15:45:55 nixos systemd[1]: Stopping Name Service Cache Daemon (nsncd)...
Jul 18 15:45:55 nixos systemd[1]: nscd.service: Deactivated successfully.
Jul 18 15:45:55 nixos systemd[1]: Stopped Name Service Cache Daemon (nsncd).
Jul 18 15:45:55 nixos systemd[1]: nscd.service: Consumed 36ms CPU time, received 1.7K IP traffic, sent 1.2K IP traffic.
Jul 18 15:45:55 nixos avahi-daemon[1520]: Got SIGHUP, reloading.
Jul 18 15:45:55 nixos avahi-daemon[1520]: No service file found in /etc/avahi/services.
Jul 18 15:45:55 nixos avahi-daemon[1520]: Leaving mDNS multicast group on interface wlp0s20f3.IPv4 with address 192.168.1.42.
Jul 18 15:45:55 nixos avahi-daemon[1520]: Interface wlp0s20f3.IPv4 no longer relevant for mDNS.
Jul 18 15:45:56 nixos systemd[1]: Starting Name Service Cache Daemon (nsncd)...
Jul 18 15:45:56 nixos systemd[1]: Started Name Service Cache Daemon (nsncd).
Jul 18 15:45:56 nixos systemd[1]: Reached target Host and Network Name Lookups.
Jul 18 15:45:56 nixos nsncd[13214]: Jul 18 12:45:56.023 INFO started, config: Config { ignored_request_types: {}, worker_count: 8, handoff_timeout: 3s }, path: "/var/run/nscd/socket"
Jul 18 15:45:56 nixos systemd[1]: Reached target User and Group Name Lookups.
Jul 18 15:45:56 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:45:56 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:45:56 nixos wpa_supplicant[1754]: p2p-dev-wlp0s20: CTRL-EVENT-DSCP-POLICY clear_all
Jul 18 15:45:56 nixos wpa_supplicant[1754]: p2p-dev-wlp0s20: CTRL-EVENT-DSCP-POLICY clear_all
Jul 18 15:45:56 nixos wpa_supplicant[1754]: nl80211: deinit ifname=p2p-dev-wlp0s20 disabled_11b_rates=0
Jul 18 15:45:56 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-DSCP-POLICY clear_all
Jul 18 15:45:56 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-DSCP-POLICY clear_all
Jul 18 15:45:56 nixos wpa_supplicant[1754]: nl80211: deinit ifname=wlp0s20f3 disabled_11b_rates=0
Jul 18 15:45:57 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:45:57 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:45:57 nixos systemd[1]: Starting NVIDIA system suspend actions...
Jul 18 15:45:57 nixos systemd[1]: Starting Pre-Sleep Actions...
Jul 18 15:45:57 nixos systemd[1]: pre-sleep.service: Deactivated successfully.
Jul 18 15:45:57 nixos systemd[1]: Finished Pre-Sleep Actions.
Jul 18 15:45:57 nixos systemd[1]: Reached target Sleep.
Jul 18 15:45:59 nixos kernel: rfkill: input handler enabled
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/ldac
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/aptx_hd
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: Unable to get default source
Jul 18 15:45:59 nixos .gsd-media-keys[2071]: Unable to get default sink
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx_hd
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/aptx
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/aac
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aac
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/sbc
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/sbc
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/sbc_xq
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/sbc_xq
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx_ll_1
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx_ll_0
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx_ll_duplex_1
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/aptx_ll_duplex_0
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/faststream
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/faststream_duplex
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/opus_05
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/opus_05
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSink/opus_05_duplex
Jul 18 15:45:59 nixos bluetoothd[1521]: Endpoint unregistered: sender=:1.152 path=/MediaEndpoint/A2DPSource/opus_05_duplex
Jul 18 15:46:00 nixos systemd[1]: nvidia-suspend.service: Deactivated successfully.
Jul 18 15:46:00 nixos systemd[1]: Finished NVIDIA system suspend actions.
Jul 18 15:46:00 nixos systemd[1]: nvidia-suspend.service: Consumed 1.022s CPU time, no IP traffic.
Jul 18 15:46:00 nixos systemd[1]: Starting System Suspend...
Jul 18 15:46:00 nixos systemd-sleep[13300]: Performing sleep operation 'suspend'...
Jul 18 15:46:00 nixos kernel: PM: suspend entry (deep)
Jul 18 15:46:00 nixos kernel: Filesystems sync: 0.026 seconds
Jul 18 15:46:37 nixos kernel: Freezing user space processes
Jul 18 15:46:37 nixos kernel: Freezing user space processes completed (elapsed 0.002 seconds)
Jul 18 15:46:37 nixos kernel: OOM killer disabled.
Jul 18 15:46:37 nixos kernel: Freezing remaining freezable tasks
Jul 18 15:46:37 nixos kernel: Freezing remaining freezable tasks completed (elapsed 0.000 seconds)
Jul 18 15:46:37 nixos kernel: printk: Suspending console(s) (use no_console_suspend to debug)
Jul 18 15:46:37 nixos kernel: ACPI: EC: interrupt blocked
Jul 18 15:46:37 nixos kernel: ACPI: PM: Preparing to enter system sleep state S3
Jul 18 15:46:37 nixos kernel: ACPI: EC: event blocked
Jul 18 15:46:37 nixos kernel: ACPI: EC: EC stopped
Jul 18 15:46:37 nixos kernel: ACPI: PM: Saving platform NVS memory
Jul 18 15:46:37 nixos kernel: Disabling non-boot CPUs ...
Jul 18 15:46:37 nixos kernel: smpboot: CPU 1 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 2 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 3 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 4 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 5 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 6 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 7 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 8 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 9 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 10 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 11 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 12 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 13 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 14 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 15 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 16 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 17 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 18 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 19 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 20 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 21 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 22 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 23 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 24 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 25 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 26 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 27 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 28 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 29 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 30 is now offline
Jul 18 15:46:37 nixos kernel: smpboot: CPU 31 is now offline
Jul 18 15:46:37 nixos kernel: ACPI: PM: Low-level resume complete
Jul 18 15:46:37 nixos kernel: ACPI: EC: EC started
Jul 18 15:46:37 nixos kernel: ACPI: PM: Restoring platform NVS memory
Jul 18 15:46:37 nixos kernel: Enabling non-boot CPUs ...
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 1 APIC 0x1
Jul 18 15:46:37 nixos kernel: CPU1 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 2 APIC 0x8
Jul 18 15:46:37 nixos kernel: CPU2 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 3 APIC 0x9
Jul 18 15:46:37 nixos kernel: CPU3 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 4 APIC 0x10
Jul 18 15:46:37 nixos kernel: CPU4 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 5 APIC 0x11
Jul 18 15:46:37 nixos kernel: CPU5 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 6 APIC 0x18
Jul 18 15:46:37 nixos kernel: CPU6 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 7 APIC 0x19
Jul 18 15:46:37 nixos kernel: CPU7 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 8 APIC 0x20
Jul 18 15:46:37 nixos kernel: CPU8 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 9 APIC 0x21
Jul 18 15:46:37 nixos kernel: CPU9 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 10 APIC 0x28
Jul 18 15:46:37 nixos kernel: CPU10 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 11 APIC 0x29
Jul 18 15:46:37 nixos kernel: CPU11 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 12 APIC 0x30
Jul 18 15:46:37 nixos kernel: CPU12 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 13 APIC 0x31
Jul 18 15:46:37 nixos kernel: CPU13 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 14 APIC 0x38
Jul 18 15:46:37 nixos kernel: CPU14 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 15 APIC 0x39
Jul 18 15:46:37 nixos kernel: CPU15 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 16 APIC 0x40
Jul 18 15:46:37 nixos kernel: core: cpu_atom PMU driver: PEBS-via-PT 
Jul 18 15:46:37 nixos kernel: ... version:                5
Jul 18 15:46:37 nixos kernel: ... bit width:              48
Jul 18 15:46:37 nixos kernel: ... generic registers:      6
Jul 18 15:46:37 nixos kernel: ... value mask:             0000ffffffffffff
Jul 18 15:46:37 nixos kernel: ... max period:             00007fffffffffff
Jul 18 15:46:37 nixos kernel: ... fixed-purpose events:   3
Jul 18 15:46:37 nixos kernel: ... event mask:             000000070000003f
Jul 18 15:46:37 nixos kernel: CPU16 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 17 APIC 0x42
Jul 18 15:46:37 nixos kernel: CPU17 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 18 APIC 0x44
Jul 18 15:46:37 nixos kernel: CPU18 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 19 APIC 0x46
Jul 18 15:46:37 nixos kernel: CPU19 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 20 APIC 0x48
Jul 18 15:46:37 nixos kernel: CPU20 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 21 APIC 0x4a
Jul 18 15:46:37 nixos kernel: CPU21 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 22 APIC 0x4c
Jul 18 15:46:37 nixos kernel: CPU22 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 23 APIC 0x4e
Jul 18 15:46:37 nixos kernel: CPU23 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 24 APIC 0x50
Jul 18 15:46:37 nixos kernel: CPU24 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 25 APIC 0x52
Jul 18 15:46:37 nixos kernel: CPU25 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 26 APIC 0x54
Jul 18 15:46:37 nixos kernel: CPU26 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 27 APIC 0x56
Jul 18 15:46:37 nixos kernel: CPU27 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 28 APIC 0x58
Jul 18 15:46:37 nixos kernel: CPU28 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 29 APIC 0x5a
Jul 18 15:46:37 nixos kernel: CPU29 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 30 APIC 0x5c
Jul 18 15:46:37 nixos kernel: CPU30 is up
Jul 18 15:46:37 nixos kernel: smpboot: Booting Node 0 Processor 31 APIC 0x5e
Jul 18 15:46:37 nixos kernel: CPU31 is up
Jul 18 15:46:37 nixos kernel: ACPI: PM: Waking up from system sleep state S3
Jul 18 15:46:37 nixos kernel: ACPI: EC: interrupt unblocked
Jul 18 15:46:37 nixos kernel: ACPI: EC: event unblocked
Jul 18 15:46:37 nixos kernel: legion PNP0C09:00: Resumed PM in legion-laptop
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: GuC firmware i915/tgl_guc_70.bin version 70.20.0
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: HuC firmware i915/tgl_huc.bin version 7.9.3
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: WRT: Invalid buffer destination
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: HuC: authenticated for all workloads
Jul 18 15:46:37 nixos kernel: nvme nvme1: D3 entry latency set to 10 seconds
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: GUC: submission enabled
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: GUC: SLPC enabled
Jul 18 15:46:37 nixos kernel: i915 0000:00:02.0: [drm] GT0: GUC: RC enabled
Jul 18 15:46:37 nixos kernel: nvme nvme1: 16/0/0 default/read/poll queues
Jul 18 15:46:37 nixos kernel: xhci_hcd 0000:3d:00.0: xHC error in resume, USBSTS 0x401, Reinit
Jul 18 15:46:37 nixos kernel: usb usb3: root hub lost power or was reset
Jul 18 15:46:37 nixos kernel: usb usb4: root hub lost power or was reset
Jul 18 15:46:37 nixos kernel: nvme nvme0: 32/0/0 default/read/poll queues
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: WFPM_UMAC_PD_NOTIFICATION: 0x20
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: WFPM_LMAC2_PD_NOTIFICATION: 0x1f
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: WFPM_AUTH_KEY_0: 0x90
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: CNVI_SCU_SEQ_DATA_DW9: 0x0
Jul 18 15:46:37 nixos kernel: iwlwifi 0000:00:14.3: RFIm is deactivated, reason = 5
Jul 18 15:46:37 nixos kernel: usb 1-11: reset high-speed USB device number 4 using xhci_hcd
Jul 18 15:46:37 nixos kernel: mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_ops [i915])
Jul 18 15:46:37 nixos kernel: pci_bus 0000:09: Allocating resources
Jul 18 15:46:37 nixos kernel: pci_bus 0000:3d: Allocating resources
Jul 18 15:46:37 nixos kernel: pci_bus 0000:09: Allocating resources
Jul 18 15:46:37 nixos kernel: pci_bus 0000:3d: Allocating resources
Jul 18 15:46:37 nixos kernel: OOM killer enabled.
Jul 18 15:46:37 nixos kernel: Restarting tasks ... done.
Jul 18 15:46:37 nixos kernel: random: crng reseeded on system resumption
Jul 18 15:46:37 nixos dockerd[2853]: time="2024-07-18T15:46:37.839443349+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t AAAA" error="read udp 172.18.0.8:55292->1.1.1.1:53: i/o timeout"
Jul 18 15:46:37 nixos dockerd[2853]: time="2024-07-18T15:46:37.839487993+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:56556->1.1.1.1:53: i/o timeout"
Jul 18 15:46:37 nixos systemd-sleep[13300]: System returned from sleep operation 'suspend'.
Jul 18 15:46:37 nixos kernel: PM: suspend exit
Jul 18 15:46:37 nixos systemd[1]: systemd-suspend.service: Deactivated successfully.
Jul 18 15:46:37 nixos systemd[1]: Finished System Suspend.
Jul 18 15:46:37 nixos systemd[1]: Stopped target Sleep.
Jul 18 15:46:37 nixos systemd[1]: Reached target Suspend.
Jul 18 15:46:37 nixos systemd-logind[1584]: Operation 'suspend' finished.
Jul 18 15:46:37 nixos ModemManager[2879]: <msg> [sleep-monitor-systemd] system is resuming
Jul 18 15:46:37 nixos systemd[1]: Starting NVIDIA system resume actions...
Jul 18 15:46:37 nixos systemd[1]: Starting Post-Resume Actions...
Jul 18 15:46:37 nixos systemd[1]: Stopped target Suspend.
Jul 18 15:46:37 nixos kernel: Generic FE-GE Realtek PHY r8169-0-7600:00: attached PHY driver (mii_bus:phy_addr=r8169-0-7600:00, irq=MAC)
Jul 18 15:46:37 nixos systemd[1]: post-resume.service: Deactivated successfully.
Jul 18 15:46:37 nixos systemd[1]: Finished Post-Resume Actions.
Jul 18 15:46:37 nixos systemd[1]: Reached target Post-Resume Actions.
Jul 18 15:46:37 nixos systemd[1]: Stopped target Post-Resume Actions.
Jul 18 15:46:38 nixos kernel: r8169 0000:76:00.0 enp118s0: Link is Down
Jul 18 15:46:38 nixos gdm[1778]: GLib: Source ID 61 was not found when attempting to remove it
Jul 18 15:46:38 nixos systemd[1]: nvidia-resume.service: Deactivated successfully.
Jul 18 15:46:38 nixos systemd[1]: Finished NVIDIA system resume actions.
Jul 18 15:46:38 nixos kernel: rfkill: input handler disabled
Jul 18 15:46:38 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:46:38 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:46:38 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:46:38 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:46:38 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:46:38 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:46:38 nixos gnome-shell[13397]: The XKEYBOARD keymap compiler (xkbcomp) reports:
Jul 18 15:46:38 nixos gnome-shell[13397]: > Warning:          Unsupported maximum keycode 708, clipping.
Jul 18 15:46:38 nixos gnome-shell[13397]: >                   X11 cannot support keycodes above 255.
Jul 18 15:46:38 nixos gnome-shell[13397]: > Warning:          Could not resolve keysym XF86KbdInputAssistPrevgrou
Jul 18 15:46:38 nixos gnome-shell[13397]: > Warning:          Could not resolve keysym XF86KbdInputAssistNextgrou
Jul 18 15:46:38 nixos gnome-shell[13397]: Errors from xkbcomp are not fatal to the X server
Jul 18 15:46:38 nixos wireplumber[2322]: spa.bluez5.midi: org.bluez.GattManager1.RegisterApplication() failed: GDBus.Error:org.bluez.Error.AlreadyExists: Already Exists
Jul 18 15:46:38 nixos bluetoothd[1521]: src/gatt-database.c:database_add_chrc() Failed to create characteristic entry in database
Jul 18 15:46:38 nixos bluetoothd[1521]: src/gatt-database.c:database_add_service() Failed to add characteristic
Jul 18 15:46:38 nixos bluetoothd[1521]: src/gatt-database.c:database_add_app() Failed to add service
Jul 18 15:46:38 nixos bluetoothd[1521]: src/gatt-database.c:client_ready_cb() Failed to create GATT service entry in local database
Jul 18 15:46:38 nixos wireplumber[2322]: spa.bluez5.midi.server: org.bluez.GattManager1.RegisterApplication() failed: GDBus.Error:org.bluez.Error.Failed: Failed to create entry in database
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/ldac
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/aptx_hd
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx_hd
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/aptx
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/aac
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aac
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/sbc
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/sbc
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/sbc_xq
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/sbc_xq
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx_ll_1
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx_ll_0
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx_ll_duplex_1
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/aptx_ll_duplex_0
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/faststream
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/faststream_duplex
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/opus_05
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/opus_05
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSink/opus_05_duplex
Jul 18 15:46:38 nixos bluetoothd[1521]: Endpoint registered: sender=:1.158 path=/MediaEndpoint/A2DPSource/opus_05_duplex
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: Unable to get default sink
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: gvc_mixer_card_get_index: assertion 'GVC_IS_MIXER_CARD (card)' failed
Jul 18 15:46:38 nixos .gsd-media-keys[2071]: Unable to get default sink
Jul 18 15:46:39 nixos systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-REGDOM-CHANGE init=DRIVER type=COUNTRY alpha2=DE
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: WPS-CANCEL
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: SME: Trying to authenticate with xxxxxxx
Jul 18 15:46:41 nixos kernel: wlp0s20f3: authenticate with xxxxxxx
Jul 18 15:46:41 nixos kernel: wlp0s20f3: send auth to xxxxxxx
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: Trying to associate with xxxxxxx
Jul 18 15:46:41 nixos kernel: wlp0s20f3: authenticated
Jul 18 15:46:41 nixos kernel: wlp0s20f3: associate with (xx:xx:xx:xx:xx:xx) (try 1/3)
Jul 18 15:46:41 nixos kernel: wlp0s20f3: RX AssocResp from xx:xx:xx:xx:xx:xx (capab=0x1911 status=0 aid=3)
Jul 18 15:46:41 nixos kernel: wlp0s20f3: associated
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: Associated with(xx:xx:xx:xx:xx:xx)
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-SUBNET-STATUS-UPDATE status=0
Jul 18 15:46:41 nixos kernel: iwlwifi 0000:00:14.3: Unhandled alg: 0x707
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: WPA: Key negotiation completed with (xx:xx:xx:xx:xx:xx) [PTK=CCMP GTK=CCMP]
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-CONNECTED - Connection to xxxxxxx completed [id=0 id_str=]
Jul 18 15:46:41 nixos dockerd[2853]: time="2024-07-18T15:46:41.840649431+03:00" level=error msg="[resolver] failed to query DNS server: 1.0.0.1:53, query: ;queue.amazonaws.com.\tIN\t AAAA" error="read udp 172.18.0.8:47793->1.0.0.1:53: i/o timeout"
Jul 18 15:46:41 nixos dockerd[2853]: time="2024-07-18T15:46:41.840718768+03:00" level=error msg="[resolver] failed to query DNS server: 1.0.0.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:60111->1.0.0.1:53: i/o timeout"
Jul 18 15:46:41 nixos dockerd[2853]: time="2024-07-18T15:46:41.840811907+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t AAAA" error="read udp 172.18.0.8:39891->1.1.1.1:53: i/o timeout"
Jul 18 15:46:41 nixos dockerd[2853]: time="2024-07-18T15:46:41.840885666+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:54637->1.1.1.1:53: i/o timeout"
Jul 18 15:46:41 nixos wpa_supplicant[1754]: wlp0s20f3: CTRL-EVENT-SIGNAL-CHANGE above=1 signal=-60 noise=9999 txrate=26000
Jul 18 15:46:41 nixos ModemManager[2879]: <msg> [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:14.3': not supported by any plugin
Jul 18 15:46:41 nixos ModemManager[2879]: <msg> [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:1b.5/0000:76:00.0': not supported by any plugin
Jul 18 15:46:42 nixos systemd[1]: Stopped target Host and Network Name Lookups.
Jul 18 15:46:42 nixos systemd[1]: Stopping Host and Network Name Lookups...
Jul 18 15:46:42 nixos systemd[1]: Stopped target User and Group Name Lookups.
Jul 18 15:46:42 nixos systemd[1]: Stopping User and Group Name Lookups...
Jul 18 15:46:42 nixos systemd[1]: Stopping Name Service Cache Daemon (nsncd)...
Jul 18 15:46:42 nixos systemd[1]: nscd.service: Deactivated successfully.
Jul 18 15:46:42 nixos systemd[1]: Stopped Name Service Cache Daemon (nsncd).
Jul 18 15:46:42 nixos avahi-daemon[1520]: Got SIGHUP, reloading.
Jul 18 15:46:42 nixos avahi-daemon[1520]: No service file found in /etc/avahi/services.
Jul 18 15:46:42 nixos avahi-daemon[1520]: Joining mDNS multicast group on interface wlp0s20f3.IPv4 with address 192.168.1.42.
Jul 18 15:46:42 nixos avahi-daemon[1520]: New relevant interface wlp0s20f3.IPv4 for mDNS.
Jul 18 15:46:42 nixos dbus-daemon[1528]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.11' (uid=0 pid=1675 comm="/nix/store/3kimhm61nra85879sxqqg7ilsm200zwi-networ" label="kernel")
Jul 18 15:46:42 nixos systemd[1]: Starting Name Service Cache Daemon (nsncd)...
Jul 18 15:46:42 nixos systemd[1]: Starting Network Manager Script Dispatcher Service...
Jul 18 15:46:42 nixos nsncd[13466]: Jul 18 12:46:42.159 INFO started, config: Config { ignored_request_types: {}, worker_count: 8, handoff_timeout: 3s }, path: "/var/run/nscd/socket"
Jul 18 15:46:42 nixos systemd[1]: Started Name Service Cache Daemon (nsncd).
Jul 18 15:46:42 nixos systemd[1]: Reached target Host and Network Name Lookups.
Jul 18 15:46:42 nixos systemd[1]: Reached target User and Group Name Lookups.
Jul 18 15:46:42 nixos dbus-daemon[1528]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Jul 18 15:46:42 nixos systemd[1]: Started Network Manager Script Dispatcher Service.
Jul 18 15:46:42 nixos gdm-password][13412]: gkr-pam: unlocked login keyring
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Contacts.SearchProvider' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Nautilus' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Calculator.SearchProvider' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Characters' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.clocks' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.seahorse.Application' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Settings.SearchProvider' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.11' (uid=1000 pid=1885 comm="/nix/store/dsikac26kpq0l2w2l1f3vjq3m7shkzqw-gnome-" label="kernel")
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Calculator.SearchProvider'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Settings.SearchProvider'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.seahorse.Application'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.clocks'
Jul 18 15:46:43 nixos .nautilus-wrapp[13515]: Connecting to org.freedesktop.Tracker3.Miner.Files
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Nautilus'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Contacts.SearchProvider'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Characters'
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.Epiphany.SearchProvider'
Jul 18 15:46:43 nixos .org.gnome.Char[13519]: JS LOG: Characters Application started
Jul 18 15:46:43 nixos .nautilus-wrapp[13515]: Unable to get contents of the bookmarks file: Error opening file /home/magpi/.gtk-bookmarks: No such file or directory
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Activating service name='org.gnome.NautilusPreviewer' requested by ':1.204' (uid=1000 pid=13515 comm="/nix/store/jify0j36lhhp9kz91y1px9p662lkh08w-nautil" label="kernel")
Jul 18 15:46:43 nixos dockerd[2853]: time="2024-07-18T15:46:43.589414115+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t AAAA" error="read udp 172.18.0.8:50625->1.1.1.1:53: i/o timeout"
Jul 18 15:46:43 nixos dockerd[2853]: time="2024-07-18T15:46:43.589443351+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:50989->1.1.1.1:53: i/o timeout"
Jul 18 15:46:43 nixos dbus-daemon[1818]: [session uid=1000 pid=1818] Successfully activated service 'org.gnome.NautilusPreviewer'
Jul 18 15:46:44 nixos systemd[1809]: Started VTE child process 13697 launched by kgx process 8835.
Jul 18 15:46:45 nixos dockerd[2853]: time="2024-07-18T15:46:45.841386580+03:00" level=error msg="[resolver] failed to query DNS server: 1.0.0.1:53, query: ;queue.amazonaws.com.\tIN\t A" error="read udp 172.18.0.8:37128->1.0.0.1:53: i/o timeout"
Jul 18 15:46:45 nixos dockerd[2853]: time="2024-07-18T15:46:45.841498941+03:00" level=error msg="[resolver] failed to query DNS server: 1.0.0.1:53, query: ;queue.amazonaws.com.\tIN\t AAAA" error="read udp 172.18.0.8:49072->1.0.0.1:53: i/o timeout"
Jul 18 15:46:45 nixos dockerd[2853]: time="2024-07-18T15:46:45.841581084+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.home.\tIN\t AAAA" error="read udp 172.18.0.8:53126->1.1.1.1:53: i/o timeout"
Jul 18 15:46:45 nixos dockerd[2853]: time="2024-07-18T15:46:45.841637926+03:00" level=error msg="[resolver] failed to query DNS server: 1.1.1.1:53, query: ;queue.amazonaws.com.home.\tIN\t A" error="read udp 172.18.0.8:55130->1.1.1.1:53: i/o timeout"
Jul 18 15:46:52 nixos systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Jul 18 15:46:53 nixos .org.gnome.Char[13519]: JS LOG: Characters Application exiting
Jul 18 15:46:57 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:46:57 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
Jul 18 15:46:58 nixos rtkit-daemon[1919]: Recovering from system lockup, not allowing further RT threads.
Jul 18 15:46:58 nixos .gnome-shell-wr[1885]: Failed to make thread 'KMS thread' normally scheduled: Operation not permitted
```

**Comment by 4JX** [comment]
Chiming in to say that I've forever had this as well on my 16ACH6H. I think this may be a "general for legion/lenovo devices" issue. Never really dived into why it happens, though.

**Comment by felipelalli** [comment]
Same issue here with my Lenovo Legion Pro 7 16IRX9H.
